lms/templates/fields/field_link_account.underscore

```diff
@@ -0,0 +1,8 @@
+<div class="u-field-value field">
+    <label class="u-field-title field-label" for="field-label-<%- id %>-default"><%- title %></label>
```

Comment:
Sorry about this. Since the corresponding element is an anchor tag so I will change it to `span`.